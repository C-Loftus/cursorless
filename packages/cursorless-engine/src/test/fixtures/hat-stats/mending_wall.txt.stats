nTokens: 479

tokenSpark:
	▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▃▁▃▁▃▃▁▁▁▁▁▃▃▁▃▃▃▃▃▃▃▃▃▃▁▃▁▁▃▃▁▃▃▁▃▃▃▃▃▃▃▁▃
	▃▃▃▃▃▃▃▃▁▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃
	▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃
	▃▃▅▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▁▃▃▃▃▃▃▃▃▃▃▅▃▁▃▃▃▃▃▃▃▃▃▃▅▃▃▃▃▅▃▃▃▃▃▃▃▃▃▃▃
	▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▅▃▃▃▅▃▃▃▃▃▃▃▃▃▃▃▃▃▅▃▃▃▃▃▅▃▃▅▃▃▅▃
	▃▃▅▃▃▃▅▃▃▃▃▁▃▃▅▅▃▅▅▃▃▃▅▅▅▃▃▃▃▅▃▅▅▅▃▅▃▅▃▅▅▅▃▃▅▃▃▃▅▅▃▅▅▃▅▃▃▃▃▅
	▅▅▅▃▅▃▅▅▅▅▅▅▅▅▃▅▃▃▃▃▃▅▅▅▃▅▃▅▅▃▁▅▅▅▅▅▅▅▅▅▃▃▅▅▅▅▅▃▅▅▅▃▃▅▅▅▅▅▅▃
	▅▅▃▅▃▅▅▅▅▅▃▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅▅

nHats:
	mean: 100%
	std: 0%
	min: 100%
	max: 100%
	spark: 0              25               50               75             █ 100

nPenalty0:
	mean: 8%
	std: 0%
	min: 7%
	max: 8%
	spark: 0    █         25               50               75               100

nPenalty1:
	mean: 64%
	std: 0%
	min: 64%
	max: 65%
	spark: 0              25               50       ▂█      75               100

nPenalty2:
	mean: 28%
	std: 0%
	min: 28%
	max: 29%
	spark: 0              25  ██           50               75               100
