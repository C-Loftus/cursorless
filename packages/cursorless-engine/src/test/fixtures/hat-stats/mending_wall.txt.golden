_       _
Mending Wall
[-----] [--]

_  _      _    _ _    __      _     _
by Robert Frost, 1914 (public domain)
[] [----] [---]| [--] |[----] [----]|

     _    _     _    _   _   _A   _  A   _ A
Something there is that doesn’t love a wall,
[-------] [---] [] [--] [---]|| [--] | [--]|

 A   _       _    _  _   _  A A     _    A B
That sends the frozen-ground-swell under it,
[--] [---] [-] [----]|[----]|[---] [---] []|

B    A      B  A     A         A B   A  _
And spills the upper boulders in the sun;
[-] [----] [-] [---] [------] [] [-] [-]|

C     _   _     A    B  _   B     B     _
And makes gaps even two can pass abreast.
[-] [---] [--] [--] [-] [-] [--] [-----]|

 C     A  A  B      B  D           A_
The work of hunters is another thing:
[-] [--] [] [-----] [] [-----] [---]|

C   B    A   B     D     A B      C
I have come after them and made repair
| [--] [--] [---] [--] [-] [--] [----]

C        _   C  A     A    A B     B  E C    C
Where they have left not one stone on a stone,
[---] [--] [--] [--] [-] [-] [---] [] | [---]|

C      A     B   D  C       D  C    C      BD
But they would have the rabbit out of hiding,
[-] [--] [---] [--] [-] [----] [-] [] [----]|

D  D       E  B       C   A E    F   E C   E
To please the yelping dogs. The gaps I mean,
[] [----] [-] [-----] [--]| [-] [--] | [--]|

 D   B F    C   F      D   A 1     1     1  F
No one has seen them made or heard them made,
[] [-] [-] [--] [--] [--] [] [---] [--] [--]|

D   2   E     D      B2    D  D     2      B B
But at spring mending-time we find them there.
[-] [] [----] [-----]|[--] [] [--] [--] [---]|

F B    C       E  B         E 3     C A
I let my neighbor know beyond the hill;
| [-] [] [------] [--] [----] [-] [--]|

3   F  4   D E  E     1    C  3   1
And on a day we meet to walk the line
[-] [] | [-] [] [--] [] [--] [-] [--]

  F D    4  F    E       C    A   C   C
And set the wall between us once again.
[-] [-] [-] [--] [-----] [] [--] [---]|

1     F  5    D  F       D  5  2   2D
We keep the wall between us as we go.
[] [--] [-] [--] [-----] [] [] [] []|

4    B  5         C    6    E  E       3   C E
To each the boulders that have fallen to each.
[] [--] [-] [------] [--] [--] [----] [] [--]|

  1   F   D     F    B    1  E       E 1
And some are loaves and some so nearly balls
[-] [--] [-] [----] [-] [--] [] [----] [---]

               A
3  6    6  E   1  1     4   D    D       D A
We have to use a spell to make them balance:
[] [--] [] [-] | [---] [] [--] [--] [-----]|

A   F    E    F  F     2  5      E    E      2_B
‘Stay where you are until our backs are turned!’
|[--] [---] [-] [-] [---] [-] [---] [-] [----]||

4     1 6      D      1   5        E       2F
We wear our fingers rough with handling them.
[] [--] [-] [-----] [---] [--] [------] [--]|

A
1 1 _          2 F     F  2 C   3 E   2
Oh, just another kind of out-door game,
[]| [--] [-----] [--] [] [-]|[--] [--]|

       A                A
 C   D 2 F   1 3  E     1  F        4 B
One on a side. It comes to little more:
[-] [] | [--]| [] [---] [] [----] [--]|

  F   6     4   1  1 3  E    2     3   1 C
There where it is we do not need the wall:
[---] [---] [] [] [] [] [-] [--] [-] [--]|

 4 5   2  2      4 6  3    3    F    2
He is all pine and I am apple orchard.
[] [] [-] [--] [-] | [] [---] [-----]|

 1  3        2   4    1   F    1
My apple trees will never get across
[] [---] [---] [--] [---] [-] [----]

                                   A
  5 5     6 2     3       3 4    3 1   5    43
And eat the cones under his pines, I tell him.
[-] [-] [-] [---] [---] [-] [---]| | [--] [-]|

 A
 1    2 4   4 C1    1        1  2         2   4D
He only says, ‘Good fences make good neighbors.’
[] [--] [--]| |[--] [----] [--] [--] [-------]||

          A                       A
 5      5 2      3      F 5 5   6 2      5
Spring is the mischief in me, and I wonder
[----] [] [-] [------] [] []| [-] | [----]

   A           A                A
 2 3   4   6   3 1       2   6  2  D
If I could put a notion in his head:
[] | [---] [-] | [----] [] [-] [--]|

      A                               A A
E  3  2    4   2  3         3   _   3F3 4
‘Why do they make good neighbors? Isn’t it
|[-] [] [--] [--] [--] [-------]| [-]|| []

      A     A             A    A     A
   6  4     4   4   A  5  1    5     5   4  5   5
Where there are cows? But here there are no cows.
[---] [---] [-] [--]| [-] [--] [---] [-] [] [--]|

       A       A A    A A      A
  3    5    6  6 1    611   3  3  5
Before I built a wall I’d ask to know
[----] | [---] | [--] ||| [-] [] [--]

 A   B   A             A
 2   1   1       4  6  1       5  6 6
What I was walling in or walling out,
[--] | [-] [-----] [] [] [-----] [-]|

  A A       B   A      B
  2 6     6 2   2   4  1    2   4     6
And to whom I was like to give offense.
[-] [] [--] | [-] [--] [] [--] [-----]|

             A  B    B  A     B      B A   A
        6    2  3    1  3    22   3  2 2   1
Something there is that doesn’t love a wall,
[-------] [---] [] [--] [---]|| [--] | [--]|

 A     A   B  A   A  B                   B    AA
 3     1   4  4   13 5 6       5 4  4  5 3    12
That wants it down.’ I could say ‘Elves’ to him,
[--] [---] [] [--]|| | [---] [-] |[---]| [] [-]|

    B  A A                A   A CAA A
4   6 63 2     5    _     3   5 116 3
But it’s not elves exactly, and I’d rather
[-] []|| [-] [---] [-----]| [-] ||| [----]

A  A     B            A C A   A   B
4  4     4 5         62 2 5   5   5
He said it for himself. I see him there
[] [--] [] [-] [-----]| | [-] [-] [---]

         B   A       A              A   A
5        3   4       1        6 6   6   5
Bringing a stone grasped firmly by the top
[------] | [---] [-----] [----] [] [-] [-]

 A   A     BA      B  A   A             A   A
 3   1     14   5  4  6  D6       6     4   3
In each hand, like an old-stone savage armed.
[] [--] [--]| [--] [] [-]|[---] [----] [---]|

B   B    C           B   B    A   B A A
1   1    3     6     5   6    2   2 3 5
He moves in darkness as it seems to me,
[] [---] [] [------] [] [] [---] [] []|

A    A A        A B    B  B      A  A   A
4    1 3        1 6    2  1      2  5   4
Not of woods only and the shade of trees.
[-] [] [---] [--] [-] [-] [---] [] [---]|

B   C   A   A  A        B  C    AB   A   A
3   4   5   1  1        2  1    23   2   6
He will not go behind his father’s saying,
[] [--] [-] [] [----] [-] [----]|| [----]|

  B B    A     A       A     A C  B  A
  2 4    1     1       1     3 5  4  4
And he likes having thought of it so well
[-] [] [---] [----] [-----] [] [] [] [--]

B    A      AB A   B    A   A    A         A   AA
5    3      61 3   3    2   4    2         2   54
He says again, ‘Good fences make good neighbors.’
[] [--] [---]| |[--] [----] [--] [--] [-------]||
